scrapers/scraper.py

```
@@ -0,0 +1,424 @@
+def get_towers(key="Tower"):
+    global lines, use_auto
+    
+    for area in areas:
+        response = requests.get(f"{root}/{area}".strip())
+        data = response.text
+        soup = BeautifulSoup(data, 'html.parser')
+        
+        for table in soup.find_all('table'):
+            headers = table.find_all('th')
+            if len(headers) > 0:
+                if headers[0].text.strip().split(" ")[0].split("[")[0] == f"{key}s":
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)